.gitignore

```diff
@@ -5,3 +5,4 @@ log
 .idea
 .vscode
 vendor
+go-ycsb
```

Comment:
Execute `make` will output the binary to `bin/go-ycsb`, seems no need to ignore the root's